tincr/io/library/genCellLibrary.tcl

```diff
@@ -1,4 +1,13 @@
-package require tincr 0.0
+package provide tincr.io.library 0.0
+package require Tcl 8.5
+package require tincr.cad.design 0.0
+package require tincr.cad.device 0.0
+package require tincr.cad.util 0.0
+
+namespace eval ::tincr:: {
+    namespace export \
+        createCellLibrary
```

Comment:
We should decide on a consistent format for function names in Tincr. When I originally wrote Tincr, I used underscores instead of camelCase (i.e. insert_cell instead of insertCell) to stay consistent with Vivado's function naming conventions. Using both will confuse users IMO.